meta-resin-common/recipes-support/resin-init/resin-init-flasher/dnsmasq.conf.systemd

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Wants=resin-net-config.service
+After=resin-net-config.service
```

Comment:
I'm a little confused by "
```
Wants=docker.service resin-net-config.service
After=docker.service resin-net-config.service
```

Why would dnsmasq depend on docker? I would expect it the other way around: docker depend on dnsmasq. @floion @petrosagg any ideas?